CHANGELOG.md

```diff
@@ -1,16 +1,113 @@
+  - Add type hints to report module ([#2445](https://github.com/MultiQC/MultiQC/pull/2445))
+  - Move `Dataset` class outside of `Plot` function, remove `Plot` dependency ([#2447](https://github.com/MultiQC/MultiQC/pull/2447))
+  - Core module refactoring: declare `mod_cust_config`, add type hints ([#2434](https://github.com/MultiQC/MultiQC/pull/2434))
+- Add `ge` and `le` to `cond_formatting_rules` ([#2494](https://github.com/MultiQC/MultiQC/pull/2494))
+- CI: use uv pip ([#2352](https://github.com/MultiQC/MultiQC/pull/2352))
+- Fix: do not attempt to render flat plot when no data ([#2490](https://github.com/MultiQC/MultiQC/pull/2490))
+- Reduce report size: exclude plot data for sections in `remove_sections` ([#2460](https://github.com/MultiQC/MultiQC/pull/2460))
+- Fix export plots with `--export` and always export data ([#2489](https://github.com/MultiQC/MultiQC/pull/2489))
+- Lint check for use of `f["content_lines"]` ([#2485](https://github.com/MultiQC/MultiQC/pull/2485))
+- Fix: make sure `modify` lambda not present in JSON dump ([#2455](https://github.com/MultiQC/MultiQC/pull/2455))
+- Consistent config options: rename `decimalPlaces` to `tt_decimals` ([#2451](https://github.com/MultiQC/MultiQC/pull/2451))
+- Enable `--export` even when writing interactive plots ([#2444](https://github.com/MultiQC/MultiQC/pull/2444))
+- Allow to set style of line graph (`lines` or `lines+markers`) per plot ([#2413](https://github.com/MultiQC/MultiQC/pull/2413))
+- Fix: replace `NaN` with `null` in exported JSON ([#2432](https://github.com/MultiQC/MultiQC/pull/2432))
+- Add `CMD` to `Dockerfile` so a default run without any parameters displays the `--help` ([#2279](https://github.com/MultiQC/MultiQC/pull/2279))
+- Remove encoding and shebang headers from module files ([#2425](https://github.com/MultiQC/MultiQC/pull/2425))
+- Fix `y_minrange` option ([#2415](https://github.com/MultiQC/MultiQC/pull/2415))
+- Refactor line plot categories: keep boolean throughout the code, and data points as pairs for simplicity ([#2418](https://github.com/MultiQC/MultiQC/pull/2418))
 
 ### New modules
 
+- [**Hostile**](https://github.com/bede/hostile) ([#2501](https://github.com/MultiQC/MultiQC/pull/2501))
+  - New module: Hostile is a short and long host reads removal tool
+- [**Sequali**](https://github.com/rhpvorderman/sequali) ([#2441](https://github.com/MultiQC/MultiQC/pull/2441))
+  - New module: Sequali Universal sequencing QC
+
+### Module updates
+
+- **FastQC**
+  - Skip per tile sequence quality section in FastQC reports for better performance ([#2552](https://github.com/MultiQC/MultiQC/pull/2552))
+  - Fix a `ZeroDivisionError` error ([#2462](https://github.com/MultiQC/MultiQC/pull/2462))
+  - Make 7 times faster and use 10 times less memory ([#2516](https://github.com/MultiQC/MultiQC/pull/2516), )
```

Comment:
This refers to the PR that only does `del self.fastqc_data` that does not seem correct with the description.

Reply:
Thanks for spotting!